airflow/ui/src/pages/TaskInstance/RenderedTemplates.tsx

```diff
@@ -0,0 +1,68 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { Box, HStack, Table } from "@chakra-ui/react";
+import { useParams, useSearchParams } from "react-router-dom";
+
+import { useTaskInstanceServiceGetMappedTaskInstance } from "openapi/queries";
+import { ClipboardRoot, ClipboardIconButton } from "src/components/ui";
+
+export const RenderedTemplates = () => {
+  const { dagId = "", runId = "", taskId = "" } = useParams();
+  const [searchParams] = useSearchParams();
+
+  const mapIndexParam = searchParams.get("map_index");
+  const mapIndex = parseInt(mapIndexParam ?? "-1", 10);
+
+  const { data: taskInstance } = useTaskInstanceServiceGetMappedTaskInstance({
+    dagId,
+    dagRunId: runId,
+    mapIndex,
+    taskId,
+  });
+
+  return (
+    <Box p={2}>
+      <Table.Root striped>
+        <Table.Body>
+          {Object.entries(taskInstance?.rendered_fields ?? {}).map(([key, value]) => {
+            if (value !== null && value !== undefined) {
+              const renderedValue: string = typeof value === "object" ? JSON.stringify(value) : String(value);
+
+              return (
+                <Table.Row key={key}>
+                  <Table.Cell>{key}</Table.Cell>
+                  <Table.Cell>
+                    <HStack>
+                      {renderedValue}
```

Comment:
```suggestion
                      <Code>{renderedValue}</Code>
```

Let's put this inside a <Code> component?

Reply:
Code would be good for the starting, then I hope that we can enhance in future to render based on type, e.g. render a dict also as object tree... I loved this in Airflow 2.10 UI. (And introduced a couple of bugs with this)

Reply:
Yes, let's definitely revive syntax highlighting after this PR.

Reply:
Thanks, done. The main thing affecting syntax highlighting is the lack of extension info for the template from the API. Once it's available it would be just picking up the correct highlighter language in the frontend part.